### ISSUE TITLE
Enable cross-origin

### ISSUE THREAD

**Issue by jhonber** [pin3da/poolparty#2]
Requirement for #1

**Comment by pin3da** [comment]
@jhonber could you check 07129cd4ab32cd29 ? ... I worked for me

**Comment by jhonber** [comment]
When I try this:

``` javascript
var urlInfo = 'https://www.youtube.com/oembed?url=' +
      'http://www.youtube.com/watch?v=' + videoId + '&format=json';
$.getJSON(urlInfo, function(info) {}
```

I get the follow error:

`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://www.youtube.com/oembed?url=http://www.youtube.com/watch?v=yywL2rc-QZM&format=json. (Reason: CORS header 'Access-Control-Allow-Origin' missing).`

**Comment by pin3da** [comment]
I tested with your code and it also works.

http://s15.postimg.org/6adb9cf8a/cors.jpg

I used iceweasel, but maybe if you are using chrome you could get problems with localhost : https://code.google.com/p/chromium/issues/detail?id=67743

**Comment by pin3da** [comment]
Solved at #3
